### PR TITLE
Treasury param

### PR DESCRIPTION
**Pull request by BurntVal**

### CODE REVIEW

File: packages/abstraxion-core/src/AbstraxionAuth.ts [hunk 1]
```diff
@@ -38,6 +39,7 @@ export class AbstraxionAuth {
    * @param {boolean} [stake] - Indicates whether staking is enabled.
    * @param {SpendLimit[]} [bank] - The spend limits for the user.
    * @param {string} callbackUrl - preferred callback url to override default
+   * @param {string} treasury - treasury contract instance
```
Comment (justinbarry) [inline packages/abstraxion-core/src/AbstraxionAuth.ts:42]: Probably should say `treasury contract address` here

**Review by justinbarry — APPROVED** [review]
LGTM

**Review by BurntNerve — APPROVED** [review]
Looks great

**Review by justinbarry — APPROVED** [review]
LGTM

**Review by justinbarry — APPROVED** [review]
LGTM

**Review by burnt-sun — APPROVED** [review]

**Pull request merged**